### ISSUE TITLE
About Python version

### ISSUE THREAD

**Issue by wangyongjie-ntu** [sbme-tutorials/liver-fibrosis-prediction-model#1]
Can anyone know what's wrong with my python code on the discretization CSV file? Thanks
The accuracy is only ~25%. Just randomly guess.  
I think there is not any distinct difference with your knn.R code. 
The same problem appears with naive-Bayes and SVM classifiers.

```
import pandas as pd
import numpy as np
from sklearn.model_selection import KFold
from sklearn.neighbors import KNeighborsClassifier
from sklearn.metrics import *

df = pd.read_csv("hcv-data-dis.csv")
y = df['Baselinehistological.staging'].to_numpy()
X = df.drop(columns = ['Baselinehistological.staging']).to_numpy()

kf = KFold(10)
model = KNeighborsClassifier(37)
for train_index, test_index in kf.split(X):
    #print("TRAIN:", train_index, "TEST:", test_index)
    X_train, X_test = X[train_index], X[test_index]
    y_train, y_test = y[train_index], y[test_index]
    model.fit(X_train, y_train)
    y_pred = model.predict(X_test)
    print(classification_report(y_pred, y_test))
```

**Comment by nguyenquivinhquang** [comment]
I have the same problem. Did you solve it?
